<launch>

  <param name="left_w0" value="1.1190389847624094"/>
  <param name="left_w1" value="1.9113400617052256"/>
  <param name="left_w2" value="2.2453643782672743"/>
  <param name="left_e0" value="-1.5615924420673513"/>
  <param name="left_e1" value="0.9618059540041545"/>
  <param name="left_s0" value="-0.7198204847152304"/>
  <param name="left_s1" value="-0.5633544443509182"/>

  <param name="left2_w0" value="1.5485536053703253"/>
  <param name="left2_w1" value="1.7909225698562206"/>
  <param name="left2_w2" value="0.2611602291374917"/>
  <param name="left2_e0" value="-1.7138400352649785"/>
  <param name="left2_e1" value="0.37735927381981177"/>
  <param name="left2_s0" value="-0.17525730501590853"/>
  <param name="left2_s1" value="-0.3286553838044499"/>

  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="1.28 0.10 0.19 0 0 0 base camera_link 1000" />

  <include file="$(find baxter_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <include file="$(find baxter_moveit_config)/launch/move_group.launch">
    <arg name="allow_trajectory_execution" value="true"/>
  </include>

  <node name="camera_control" pkg="baxter_tools" type="camera_control.py"
    args="-o left_hand_camera -r 640x400"/>

  <!-- Gripper Argument -->
  <arg name="gripper" default="right" />

  <!-- Start the Gripper Action Server -->
  <node name="rsdk_gripper_action_server" pkg="baxter_interface" type="gripper_action_server.py" args="--gripper $(arg gripper)" required="true" />

  <node pkg="baxter_interface" type="joint_trajectory_action_server.py" name="joint_traj_action_srv" output="screen"
	respawn="false" />

  <node pkg="shell_game" type="find_treasure.py" name="find_treasure" output="screen" respawn="true"/>
  <node pkg="shell_game" type="cup_tracking.py" name="image_converter" output="screen" respawn="true"/>

  <node pkg="shell_game" type="get_cup.py" name="get_cup" output="screen" respawn="false" launch-prefix="xterm -e "/>

</launch>
